### ISSUE TITLE
Feature Request: Please add ability to see distribution of expenses based on tags

### ISSUE THREAD

**Issue by KrishnenduMarathe** [mtotschnig/MyExpenses#1350]
While we can use Categories to check the distribution of expenses, I have many expenses tagged and wish I could use see some charts for these tags to gain some useful insights and better help my expense plans.

Apart from this, thanks for your hardwork. This app has been a blessing and I rely on it for my day to day exchanges. Appreciate you guys doing it for free and keeping the best possible quality and design of any budget app I have seen so far.

**Comment by KrishnenduMarathe** [comment]
No, that's not what i meant. I wished if we could chart distribution based
on tags alone, not categories. In my use case, I have created specialized
tags that I used to mark each transaction and one transaction can have
multiple tags. I wanted to visualize my expenses based on these tags. Would
this be possible?

**Comment by mtotschnig** [comment]
@KrishnenduMarathe Thanks for the suggestion. Do you have an idea how this could be best visually represented. A pie chart like the one used for the category distribution might not make sense, since the sum of all sums per tag will most likely be more than 100%.

**Comment by mtotschnig** [comment]
@KrishnenduMarathe Sorry for the late reply. I am afraid I am not understanding your suggestion. Could you point to any examples of similar designs or create some quick mockups?
